controllers/productController.js

```diff
@@ -22,6 +22,21 @@ const productsInqury = catchAsync(async (req, res) => {
   });
 });
 
+const productDetailInqury = catchAsync(async (req, res) => {
+  const { productId } = req.params;
+  if (!productId) {
+    return res.status(400).json({ message: "KEY_ERROR" });
+  }
+  const productDetailInqury = await productService.productDetailInqury(
+    productId
```

Comment:
- 변수명은 product로 충분 할 것 같습니다.
- prouctId가 DB에 없는 값이 올 때에 대한 예외 처리를 추가해 주세요. DB에 없는 데이터를 조회 하는 것이기 때문에, 404 상태코드로 예외처리 해주시면 됩니다.

```suggestion
  const product = await productService.productDetailInqury(
    productId
```